htdocs/core/modules/contract/doc/pdf_strato.modules.php

```diff
@@ -364,7 +364,7 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 							$txtpredefinedservice .= $objectligne->product_label;
 						}
 
-						$desc = dol_htmlentitiesbr($objectligne->desc, 1); // Desc (not empty for free lines)
+						$desc = dol_htmlentitiesbr($objectligne->product_desc, 1); // Desc (not empty for free lines)
```

Comment:
The $line->product_desc is the value that is used by default to fill the line when we enter a new product line. Once the line hase been recorded we should not use it anymore. Its cntent is inside $line->desc

So current code seems good.


If we used a non predefined product:
the field description of line is filled manually and is also saved into line->desc.

So the line to show is $objectline->desc.

Reply:
Ok, je viens de comprendre mon erreur.
Le contrat a été créer avant que je remplisse la description du service.
Les nouveaux contrats sont ok.
Merci et désolé pour le dérangement.